xrpl_api/src/api/tx.rs

```diff
@@ -44,16 +44,6 @@ impl TxRequest {
 
 #[derive(Debug, Deserialize)]
 pub struct TxResponse {
-    /// The SHA-512 hash of the transaction.
```

Comment:
These three fields also exist in `Transaction` so I removed them to let them be deserialized into the `Transaction` struct